pkgs/applications/editors/eclipse/plugins.nix

```diff
@@ -233,6 +233,8 @@ rec {
 
     srcs = [ srcFeature srcPlugin1 srcPlugin2 ];
 
+    bildInputs = [ zest ];
```

Comment:
Here `bildInputs` is misspelled. Also, it will be necessary to use `propagatedBuildInputs` since otherwise the `zest` dependency wouldn't show up in the Eclipse dropin directory.